src/backend/TrafficCourts/Staff.Service/Controllers/JJController.cs

```diff
@@ -35,13 +35,13 @@ public JJController(IJJDisputeService JJDisputeService, ILogger<JJController> lo
     [HttpGet("Disputes")]
     [ProducesResponseType(typeof(IList<JJDispute>), StatusCodes.Status200OK)]
     [ProducesResponseType(StatusCodes.Status500InternalServerError)]
-    public async Task<IActionResult> GetJJDisputesAsync(string? jjGroupAssignedTo, string? jjAssignedTo, CancellationToken cancellationToken)
+    public async Task<IActionResult> GetJJDisputesAsync(string? jjAssignedTo, CancellationToken cancellationToken)
```

Comment:
We need to remove this XML comment parameter tag as well if we are removing all jjGroupAssignedTo parameters.

Reply:
got it